code/game/objects/structures/flora.dm

```diff
@@ -319,37 +319,6 @@
 	desc = "A gift from the botanical staff, presented after the RD's reassignment. There's a tag on it that says \"Y'all come back now, y'hear?\"\nIt doesn't look very healthy..."
 	icon_state = "plant-dead"
 
-//a rock is flora according to where the icon file is
-//and now these defines
-/obj/structure/flora/rock
-	name = "rock"
-	desc = "A rock."
-	icon_state = "rock1"
-	icon = 'icons/obj/flora/rocks.dmi'
-	resistance_flags = FIRE_PROOF
-	anchored = TRUE
-
```

Comment:
it would appear that these are still used

Reply:
Looks like it, yeah